src/reconcile/rec_visibility.c

```diff
@@ -792,6 +791,5 @@ __wt_rec_upd_select(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_INSERT *ins, W
 
     __wt_rec_time_window_clear_obsolete(session, upd_select, NULL, r);
 err:
```

Comment:
Whenever WT_ERR() is used around a function, it means that whenever the underlying function returns a non-zero value (0 denotes successful), we go to the label `err`. Usually we only need to go to the `err` label when there we still need to clean some variables before exiting the function (e.g. freeing variables that are malloced inside the function). Since we don't have the need to free any variables anymore, would it be okay to remove the `err` label and replace it any `WT_ERR` to `WT_RET`?

Reply:
I agree with Jie here - we can replace `WT_ERR()` with `WT_RET()` and remove the `err` label.

Reply:
Thank you for pointing that out @jiechenbo ! That makes sense, I have just committed those changes.